src/main/java/taxi/controller/authentication/RegisterController.java

```diff
@@ -0,0 +1,42 @@
+package taxi.controller.authentication;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.exception.DataProcessingException;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.DriverService;
+
+public class RegisterController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/authentication/register.jsp")
+                .forward(req, resp);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException, ServletException {
+        String name = req.getParameter("name");
+        String licenseNumber = req.getParameter("license_number");
+        String login = req.getParameter("user_name");
```

Comment:
it would be better to use 
```suggestion
        String login = req.getParameter("login");
```